backend/src/schema/resolvers/passwordReset.js

```diff
@@ -34,7 +34,8 @@ export default {
   Mutation: {
     requestPasswordReset: async (_parent, { email }, { driver }) => {
       const nonce = uuid().substring(0, 6)
-      return createPasswordReset({ driver, nonce, email })
+      const lowerCasedEmail = email.toLowerCase()
+      return createPasswordReset({ driver, nonce, lowerCasedEmail })
```

Comment:
This is not going to work as `email` and not `lowerCaseEmail` gets desctructured here: https://github.com/Human-Connection/Human-Connection/blob/master/backend/src/schema/resolvers/passwordReset.js#L5

Reply:
Yep, good catch will fix that.

Reply:
https://github.com/Human-Connection/Human-Connection/pull/2118 :point_left: